### DISCUSSION TITLE
Multiple instances of EMQX5 (with an operator)

### DISCUSSION THREAD

**Discussion by klemen-df** [emqx/emqx-operator#709] [Q&A] [answered]
In the (near) future we will need to have more than one instance of EMQX5 on the same AKS cluster.

Can the same cluster contain multiple instances of EMQX? What are the requirements?

Different name, different namespace?

**Comment by Rory-Z** [comment, marked as answer]
Hi, @KlemenDanfoss the EMQX Operator support multiple instances, just use different name is ok.

**Reply by klemen-df** [reply]
Hi,

we added a second instance (CRD) and changed the name:

```
  manifest = {
    "apiVersion" = "apps.emqx.io/v2alpha1"
    "kind"       = "EMQX"
    "metadata" = {
      "name"      = name1
      "namespace" = mynamespace
    }
```

```
  manifest = {
    "apiVersion" = "apps.emqx.io/v2alpha1"
    "kind"       = "EMQX"
    "metadata" = {
      "name"      = name2
      "namespace" = mynamespace
    }
```

We have now everything (core pods) but only one controller/manager pod for both emqx's. Is this how it's meant to be?

**Reply by klemen-df** [reply]
hi, @Rory-Z , any answer on this?

**Reply by Rory-Z** [reply]
Hi, @KlemenDanfoss Sorry for delay, I just got COVID-19 at last week.
The `only one controller/manager pod ` is it the `emqx-operator-controller-manager` ? If yes, that worked.

**Reply by klemen-df** [reply]
Hi,

sorry to hear that, hope you're ok!

Yes, then all is good, just wanted to double check - thanks!

**Reply by Rory-Z** [reply]
Now I’m back to health, thank you for your concern.
